consensus/clique/clique.go

```diff
@@ -137,8 +137,9 @@ var (
 	errRecentlySigned = errors.New("recently signed")
 )
 
-// SignerFn is a signer callback function to request a header to be signed by a
-// backing account.
+// SignerFn hashes and signs the data to be signed by a backing account. It should point to the
+// accounts.Wallet.SignData method.
+// The function arguments are: current address, mimeType and data to be hashed and signed.
```

Comment:
Please add parameter names to the method if you want to provide more information, it doesn't make sense to add this line as a comment vs. params.

Reply:
OK. Done.